### DISCUSSION TITLE
Seeking freelance work in JAX

### DISCUSSION THREAD

**Discussion by cool-RR** [jax-ml/jax#23268] [General]
Hey folks!

For the last 6 months I've been learning JAX as I've been using it to run experiments for [my research](https://r.rachum.com/) at Tufts University. I'm now thinking of funding my research by doing some freelance work. It would be great if there are clients who need jobs done in JAX, because then I'd be doubling-down on my JAX skills.

I don't know whether there exists such a market for JAX freelance work. I know that much of the JAX usage is at Google and DeepMind for research, which probably means it's all done in-house. **Do you know whether there are other companies or labs that use JAX and might be interested in contracting out JAX development work?** If so I'd appreciate a connection.

About myself: I've worked as a software engineer for 12 years. Ex-Googler and PSF Fellow. [My CV](https://r.rachum.com/cv).

**Comment by Imme768** [comment]
Hello. Are you still looking for freelance work to fund your research?

**Reply by cool-RR** [reply]
Yes, please email me. (Email in CV above)

**Comment by alfercorral** [comment]
Hi @cool-RR, could you share whether you got any outcome from your search? I am also interested in similar work opportunities
